### DISCUSSION TITLE
Auto Stacker need improvement

### DISCUSSION THREAD

**Discussion by WarnDa** [OmerBenGera/WildStacker#364] [Pending Feature Requests]
### Minecraft's Version

1.16.5 Airplane

### Plugin's Version

b224

### Describe the bug

The settings
>How much time should be passed between auto-stacking? (in ticks)
> If you wish to disable the auto-stacking task, set the stack-interval to 0.
> stack-interval: 2400

Is badly done.
For example if you get two stack of 30 mobs (with a limit of stack at 50), the two stack wont merge to make a stack of 50 and a stack of 10.
Also, if you got a stack of 30 mobs, and a single mobs, sometimes the whole stack will merge into the single mob.
But it's frustrating, as the stack was in a mob farm (to reproduce them), and it stacks to a single mob outside that was near.. :/

### To Reproduce

Just try to stack mobs and do tests on them, you'll see if has issues.
(It's more visible on animals, as they are never killed, but player stack a lot of animals to create mob farms)

**Comment by OmerBenGera** [comment]
I moved your issue to the discussions as it's a feature request and not a bug.
